### PR TITLE
Qa/adding contributing style guides

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

N/A

## Description of changes being made
Adding links to the style guides in the contributing doc.

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](./CONTRIBUTING.md) for more information.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -193,10 +193,16 @@ Our markdown comes with metadata - also called "frontmatter". Here are the speci
 
 ## When a PR is ready for publishing
 
+### Style guides
+
+In general, the documentation team follows the Microsoft Style Guide. Anything that deviates from that style guide is [documented here](https://wiki.d2iq.com/display/ENG/Documentation+Style+Guide).
```
Comment (cmastr) [inline CONTRIBUTING.md:198]: Should we also reference Vale here?

Comment (vacas5) [reply CONTRIBUTING.md:198]: Also, I have issues with this link

Comment (cmastr) [inline CONTRIBUTING.md:198]:
```suggestion
In general, the documentation team uses Vale as our style guide. Anything that deviates from that style guide is decided by the docs team and implemented in Vale.
```

**Review by cmastr — APPROVED** [review]

**Review by vacas5 — APPROVED** [review]

**Review by vacas5 — CHANGES_REQUESTED** [review]
@msdolbey we might as well fix the Vale errors

File: CONTRIBUTING.md (hunk 1 above)
Comment (msdolbey) [reply CONTRIBUTING.md]: Added your comment @cmastr - more accurate, surely.

**Comment by msdolbey** [comment]
> @msdolbey we might as well fix the Vale errors

@vacas5 - good call - done.

File: .github/styles/Vocab/Docs/accept.txt [hunk 2]
```diff
@@ -38,6 +38,8 @@ GRU
 Hashicorp
 Horovod
 hostname
+hotfix
```
Comment (cmastr) [inline .github/styles/Vocab/Docs/accept.txt:41]: You can also do this like below (|es)

File: .github/styles/Vocab/Docs/accept.txt [hunk 3]
```diff
@@ -38,6 +38,8 @@ GRU
...
+hotfixes
```
Comment (msdolbey) [inline .github/styles/Vocab/Docs/accept.txt:42]:
```suggestion
hotfix(|es)
```

**Review by vacas5 — APPROVED** [review]

**Pull request merged**